README.md

```diff
@@ -163,7 +163,13 @@ So others could generate that without JVM and others could generate other target
 
 #### Installing JDK
 - Install [JDK8](http://www.oracle.com/technetwork/java/javase/downloads/)
+- Setup $JAVA_HOME
+```
+echo "export JAVA_HOME=$(/usr/libexec/java_home)" >> ~/.bash_profile
```

Comment:
I use zsh on mac. Also for example intellij (and other programs) doesnt get those environments. In mac each version has different ways for creating actual "global" envs. Now im using one related to a plist file. It is a pity that there isnt an standarized way